src/cloudant/errors.py

```diff
@@ -49,3 +49,30 @@ class CloudantArgumentError(CloudantException):
     """
     def __init__(self, msg, code=400):
         super(CloudantArgumentError, self).__init__(msg, code)
+
+class ResultException(CloudantException):
+    """
+    Provides a way to issue Cloudant Python client library result specific
+    exceptions.
+
+    :param int code: An code value used to identify the result exception.
```

Comment:
Rather than switching to `Value used to...`, I made it `A code value used to...` to keep consistent with the other two Cloudant exception classes.  Fixed in https://github.com/cloudant/python-cloudant/pull/124/commits/2bb4fc32c7434e1c22c1bd89875d5e186669d919.